force_bdss/app/evaluate_operation.py

```diff
@@ -14,6 +14,13 @@ class EvaluateOperation(BaseOperation):
     """
 
     def run(self):
+
+        """verify the workflow.
+        Do not do use this [run(do_verify=False)] on unit tests :
+        use OptimizeOperation.run() to test verification.
+        """
+        self.verify_workflow()
+
         """ Evaluate the workflow. """
```

Comment:
Style nitpick: we typically use `#` for inline comments, the triple `""" """` quote blocks are reserved for docstrings. This also becomes important for our Sphinx auto doc generation as well

If in doubt, follow the NumPy example code: https://numpydoc.readthedocs.io/en/latest/example.html#source